VotingApplication/VotingApplication.Web/Scripts/Directives/ResultsChart.html

```diff
@@ -0,0 +1,5 @@
+﻿<link href="/Content/Scss/ResultsChart.scss" rel="stylesheet">
```

Comment:
When we turn on minification, I don't think this will play nice with the bundling.